### PR TITLE
Increase -Xmx for Maven in .travis.yml

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: java
 
 env:
   global:
-    - MAVEN_OPTS="-Xmx256M"
-    - MAVEN_SKIP_CHECKS_AND_DOCS="-Dair.check.skip-all=true -Dmaven.javadoc.skip=true"
```
Comment (electrum) [inline .travis.yml]: Why this change? The documentation shows several examples of them being quoted:
```
- PACKAGE_VERSION="1.0.*"
```
```
JRUBY_OPTS="--server -Dcext.enabled=false -Xcompile.invokedynamic=false"
```
Also, should the commit message say "quote" instead of "escape"?

Comment (arhimondr) [inline .travis.yml]: We do not quote the matrix variables. So, just to keep the same here. I can remove this commit though.

**Comment by martint** [comment]
Merged, thanks!
